tinymce-angular-component/src/test/ts/browser/LoadTinyTest.ts

```diff
@@ -85,7 +85,7 @@ UnitTest.asynctest('LoadTinyTest', (success, failure) => {
     ]),
     Log.chainsAsStep('Should be able to load TinyMCE from Cloud', '', [
       cSetupEditor([ 'apiKey="a-fake-api-key"', 'cloudChannel="6-dev"' ], []),
-      cAssertTinymceVersion('6'),
```

Comment:
We might need to change the cloud channel to `6-stable` and add a new test for `7-stable`

Reply:
Now using channel `6` for this test. And I'm leaving adding the 7 load test to a different PR for specifically TinyMCE v7 support